ergo/metaculus.py

```diff
@@ -275,6 +337,20 @@ def score_prediction(self, prediction, resolution: float) -> ScoredPrediction:
             prediction["t"], prediction, resolution, score, self.__str__()
         )
 
+    def change_since(self, since: int):
```

Comment:
Why is the `since` argument type `int` if it's a timestamp?

Reply:
Will change to `float`.

Reply:
Why not `datetime`?

Reply:
I guess I was thinking a `timestamp` (which is a `float` in Python I think), partly because that's what the Metaculus API returns. I'm happy to use `datetime` instead.